signer/src/transaction_coordinator.rs

```diff
@@ -305,6 +316,28 @@ where
             };
         }
 
+        // If we have just deployed the smart contracts, trying to broadcast the
+        // key rotation tx would result in transaction rejected, reason: NoSuchContract
+        if self.sbtc_contracts_deployed && !just_deployed_contracts {
```

Comment:
QQ: What's the thought on intended behavior if this transaction gets submitted but not mined by the next bitcoin block, then the next coordinator does the same, etc.? It won't be an RBF I guess because the signatures-portion of the stacks tx isn't deterministic in our case so the txid will be new -- is the idea that it just hangs around until garbage collected? and could that cause problems with nonces?

Just thinking out loud :thinking:

Reply:
> ... is the idea that it just hangs around until garbage collected? and could that cause problems with nonces?

Hmmm, I don't think so, but I'm not sure. Generally speaking, if we have two transactions from the same account using the same nonce which have the same cost but have different fees then I think the miner will just choose the one with the greater fee. I would assume that Stacks nodes allow for transactions with the same nonce to exist in the mempool, but I don't know for sure. If the natural thing here is indeed the case (same cost, same nonce, different fees) are allowed then we should be okay.

Reply:
Yeah you're right, the code is [here](https://github.com/stacks-network/stacks-core/blob/40059a57cd27e740c5e9d91a833fb2c975b0bf0b/stackslib/src/core/mempool.rs#L2122) for RBF. Looks like it's only a check for address + nonce + fee is greater than previous. if fee isn't > last fee it will be rejected.

Reply:
In addition to what was said above, if for some strange timing reason we have two valid rotate keys tx (with different, valid nonces) then the second one will fail with `ERR_AGG_PUBKEY_REPLAY`.